src/components/EmptyState/EmptyState.stories.mdx

```diff
@@ -0,0 +1,35 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+
+import EmptyState from "./EmptyState";
+import Icon, { ICONS } from "../Icon";
+
+<Meta title="EmptyState" component={EmptyState} />
+
+### EmptyState
+
+This is a [React](https://reactjs.org/) component to represent an empty state.
+
+### Props
+
+<ArgsTable of={EmptyState} />
+
+### Default
+
+<Canvas>
+  <Story name="Default">
+    <EmptyState
+      iconName="containers"
+      title="No instances found"
+    >
+      <p>
+        There are no instances in this project. Spin up your first instance!
+      </p>
+      <p>
+        <a className="p-link--external" href="#todo" target="_blank">
```

Comment:
Just as a drive by, we don't use the external link class anymore.

Reply:
Ok, no problem, I will remove it from this example, and take a note to do the same in LXD and Anbox.